lib/Common/ConfigFlagsList.h

```diff
@@ -1477,6 +1487,9 @@ FLAGR(Number, JITServerIdleTimeout, "Idle timeout in seconds to do the cleanup i
 FLAGR(Number, JITServerMaxInactivePageAllocatorCount, "Max inactive page allocators to keep before schedule a cleanup", 10)
 
 FLAGNR(Boolean, StrictWriteBarrierCheck, "Check write barrier setting on none write barrier pages", false)
+FLAGR (Boolean, WriteBarrierTest, "Always return true while checking barrier to test recycler regardless of annotation", DEFAULT_CONFIG_WriteBarrierTest)
+FLAGR (Boolean, EnableHardwareWriteBarrier, "Use to turn off write watch to test software write barrier on windows", DEFAULT_CONFIG_EnableHardwareWriteBarrier)
+FLAGR (Boolean, EnableBGFreeZero, "Use to turn off background freeing and zeroing to simulate linux", DEFAULT_CONFIG_EnableBGFreeZero)
```

Comment:
FLAGR means available in release, no?
if you want to test in free build I think you need to change the default value and rebuild. in free build the unnecessary code is optimized. 

---
In reply to: [87881989](https://github.com/Microsoft/ChakraCore/pull/1968#discussion_r87881989) [](ancestors = 87881989)